app/models/miq_request_workflow.rb

```diff
@@ -487,6 +487,12 @@ def validate_length(_field, _values, dlg, fld, value)
     return "#{required_description(dlg, fld)} must not be greater than #{fld[:max_length]} characters" if fld[:max_length] && value.to_s.length > fld[:max_length]
   end
 
+  def validate_regex(_field, _values, dlg, fld, value)
+    regex = Regexp.new(fld[:required_regex], Regexp::IGNORECASE)
```

Comment:
The `: required_regex` field can be configured as a real Regexp object so you do not need to convert it here and it would allow the caller to define if they wanted to ignore the case or not.

I was thinking the field would be configured more like this:
```
---
:required_regex: !ruby/regexp /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
```